src/components/ConnectExchangeView/ExchangeAccountSettings/ConfirmDeleteDialog/ConfirmDeleteDialog.js

```diff
@@ -51,6 +51,8 @@ const ConfirmDeleteDialog = ({ onClose, open }) => {
     ["profitSharing"],
     true,
   );
+
+  const providerAvailable = providers && providers.length;
```

Comment:
Any reason to move this to a new variable? `providers` will never be null anyway so the `providers` check can be removed. And please use a better variable name, like `hasConnectedPS`

Reply:
Done